authors/apps/authentication/tests.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import test, status
+from .models import User
+from .jwt_generator import jwt_encode, jwt_decode
+
+
+class TestJWTGenerator(TestCase):
+    def setUp(self):
+        self.id = 11
+        self.token = jwt_encode(self.id)
+    
+    def test_that_token_encoded_decoded(self):
+        decoded_id = jwt_decode(self.token)
+
+        self.assertTrue(self.token)
+        self.assertEqual(decoded_id['user_id'], self.id)
+    
+    def test_user_token_property(self):
```

Comment:
To get a token, a user will make a request to an endpoint. Therefore, I am of the opinion that we should add a test that sends a request to the `/users/login` endpoint and we check whether it returns a token or not